vxportia/router.py

```diff
@@ -0,0 +1,92 @@
+
+from vumi.errors import DispatcherError
+from vumi.dispatchers.base import BaseDispatchRouter
```

Comment:
You need to use the things from `vumi.dispatchers.endpoint_dispatchers` because that supplies `connector_name` and the older base things don't. Using the `transport_name` is problematic because if there is a load balancer or other dispatcher connected instead of the transport being connected directly, then the transport name will not be what's expected or the correct RabbitMQ connector to send the message to.